src/services/explore.ts

```diff
@@ -49,33 +49,35 @@ export function saveFilter(filter: any, name: string, token?: string) {
     );
 }
 
-export function fetchElementsInfos(
+export async function fetchElementsInfos(
```

Comment:
Why using the async / await syntax here ? Every functions un services uses Promise format.
I'm sure it's not necessary here. A Promise.all should do the trick if it's acceptable to send multiple paralell requests and that the order of ElementsInfos doesn't matter.